src/jslib.js

```diff
@@ -393,15 +475,14 @@ const createproxy = function(L1, p, type) {
 		target = function(){};
 		break;
 	case "object":
-		target = {};
+		target = Object.create(null);
```

Comment:
Any reason to discard the default Object prototype there ?

Reply:
Good question. Various traps have requirements based on the underlying object. For function objects this was breaking things trying to run:
```js
p=new Proxy(function(){}, {ownKeys: function(t){ return [];}}); Reflect.ownKeys(p);
```
Which gives an error like:
> TypeError: proxy can't skip a non-configurable property '"prototype"'

------

However now that I try things, objects don't seem to come with any non-configurable properties. So I can probably remove this commit/change.

Reply:
Removed from PR for now.